### DISCUSSION TITLE
Edit Ticker of transactions in activities.

### DISCUSSION THREAD

**Discussion by Pindol83** [ghostfolio/ghostfolio#2576] [Q&A]
Hello everyone,

I have some issues with an import on Ghostfolio. I imported transactions from my broker, but some of them are not recognized correctly because there are cases where two companies have the same ticker. I wanted to edit and input the correct ticker in the activities, but Ghostfolio doesn't allow me to do that. Is there a way to do it?

thanks.

**Comment by dtslvr** [comment]
Hi @Pindol83

I would delete the affected activity, correct the ticker in your broker's file, and reimport into Ghostfolio.

**Reply by Pindol83** [reply]
Thanks, in the end, I did this. The problem was that the ticker GOLD (Barrick Gold) was getting confused with gold futures.
